ci/run-integration-tests.ps1

```diff
@@ -3,6 +3,8 @@ param (
     [string]$RepoName
 )
 
+$env:RESOURCE_KEY = $Keys.TestResourceKey
```

Comment:
Why is a resource key needed? This repo doesn't have device-detection or location.

Reply:
Sorry, it's mistake

Reply:
Actually key used in fiftyone.pipeline.cloudrequestengine\tests\integration.test.js

Reply:
Ah, I see. If it's used there, then that's fine.